microservices/front-end/src/App.tsx

```diff
@@ -0,0 +1,21 @@
+// import { useState } from 'react'
+// import reactLogo from './assets/react.svg'
+// import viteLogo from '/vite.svg'
+import './App.css'
+import { BrowserRouter as Router, Route, Routes, Navigate } from 'react-router-dom';
+import QuestionBank from './components/QuestionBank'
+import LoginPage from './components/LoginPage'
+
+function App() {
+  return (
+    <Router>
+      <Routes>
+        <Route path = "/login" element = {<LoginPage/>} />
+        <Route path='/question-bank' element = {<QuestionBank/> } />
+        <Route path="*" element={<Navigate to="/login" />} /> {/* Catch-all route */}
```

Comment:
Will we be penalised for assignment 1 if we do not use a single page app?

Reply:
This is not for assignment 1. It is because Stefan asked me to do a Login and Signup Page so I added this in. Later for assignment 1, we can just change the catch-all route to QuestionBank

Reply:
We can ask our mentor tmr because an app with react router can still technically be considered single page